### ISSUE TITLE
Suppress warning outputs during testing

### ISSUE THREAD

**Issue by iuliadmtru** [xKDR/Survey.jl#125]
Right now warnings that are generated by tests appear in the log messages. I think we can suppress them (since we intentionally test for them), it would be easier to read the logs.

**Comment by smishr** [comment]
Is this related to #42 ?

**Comment by smishr** [comment]
@iuliadmtru add a PR for this into `singledesign` branch.

**Comment by iuliadmtru** [comment]
This was mostly solved with #168, apart from three warnings: two coming from `sturges` and `freedman_diaconis` which I mentioned in [this comment](https://github.com/xKDR/Survey.jl/pull/168#issuecomment-1377162807) in the same PR, and one that is the last log when running `julia make.jl`:
```julia
┌ Warning: Documenter could not auto-detect the building environment Skipping deployment.
└ @ Documenter ~/.julia/packages/Documenter/48VFh/src/deployconfig.jl:75
```
I don't know how we can avoid this, or even if we can.

**Comment by smishr** [comment]
New `SurveyDesign` doesnt have warnings.
